dashboard/app/auth.go

```diff
@@ -57,10 +58,20 @@ func mkAuthEndpoint(u string) authEndpoint {
 	return authEndpoint{url: u}
 }
 
+// The JSON representaion of JWT claims.
+type jwtClaimsParse struct {
+	Subject  string `json:"sub"`
+	Audience string `json:"aud"`
+	// The field in the JSON is a string but contains a UNIX time.
+	Expiration string `json:"exp"`
+}
+
+// The typed representation of JWT claims.
 type jwtClaims struct {
-	Subject    string  `json:"sub"`
-	Expiration float64 `json:"exp"`
```

Comment:
I don't know if  you considered this or not, but one option is to have both:
```
    Expiration float64 `json:"exp"`
    expiration time.Time
```
and make parsing function fill in the second field.